<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPiano</class>
 <widget class="QMainWindow" name="VPiano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drumstick Virtual Piano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vpiano.qrc">
    <normaloff>:/drumstick.png</normaloff>:/drumstick.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="drumstick::widgets::PianoKeybd" name="pianokeybd"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionConnections"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionRaw_Computer_Keyboard"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuNames">
     <property name="title">
      <string>Show Note Names</string>
     </property>
     <addaction name="actionNever"/>
     <addaction name="actionMinimal"/>
     <addaction name="actionWhen_Activated"/>
     <addaction name="actionAlways"/>
    </widget>
    <widget class="QMenu" name="menuBlack_Keys_Names">
     <property name="title">
      <string>Black Keys Names</string>
     </property>
     <addaction name="actionSharps"/>
     <addaction name="actionFlats"/>
     <addaction name="actionNothing"/>
    </widget>
    <widget class="QMenu" name="menuNames_Orientation">
     <property name="title">
      <string>Names Orientation</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
     <addaction name="actionAutomatic"/>
    </widget>
    <widget class="QMenu" name="menuOctaves_Naming">
     <property name="title">
      <string>Central Octave C Note Name</string>
     </property>
     <addaction name="actionNoOctaves"/>
     <addaction name="actionC3"/>
     <addaction name="actionC4"/>
     <addaction name="actionC5"/>
    </widget>
    <widget class="QMenu" name="menuNote_Names">
     <property name="title">
      <string>Note Names</string>
     </property>
     <addaction name="actionStandard"/>
     <addaction name="actionCustom_Sharps"/>
     <addaction name="actionCustom_Flats"/>
    </widget>
    <addaction name="menuNames"/>
    <addaction name="menuBlack_Keys_Names"/>
    <addaction name="menuNames_Orientation"/>
    <addaction name="menuNote_Names"/>
    <addaction name="menuOctaves_Naming"/>
    <addaction name="actionNames_Font"/>
    <addaction name="actionInverted_Keys_Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionConnections">
   <property name="text">
    <string>&amp;Connections</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNever">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Never</string>
   </property>
  </action>
  <action name="actionWhen_Activated">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>When Activated</string>
   </property>
  </action>
  <action name="actionAlways">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always</string>
   </property>
  </action>
  <action name="actionSharps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sharps</string>
   </property>
  </action>
  <action name="actionFlats">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Flats</string>
   </property>
  </action>
  <action name="actionNothing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nothing</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionAutomatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionNames_Font">
   <property name="text">
    <string>Names Font</string>
   </property>
  </action>
  <action name="actionMinimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimal</string>
   </property>
  </action>
  <action name="actionC3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C3</string>
   </property>
  </action>
  <action name="actionC4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C4</string>
   </property>
  </action>
  <action name="actionC5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C5</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionCustom_Sharps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom with Sharps</string>
   </property>
  </action>
  <action name="actionCustom_Flats">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom with Flats</string>
   </property>
  </action>
  <action name="actionNoOctaves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionInverted_Keys_Color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inverted Keys Color</string>
   </property>
  </action>
  <action name="actionRaw_Computer_Keyboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw Computer Keyboard</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drumstick::widgets::PianoKeybd</class>
   <extends>QGraphicsView</extends>
   <header location="global">drumstick/pianokeybd.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vpiano.qrc"/>
 </resources>
 <connections/>
</ui>
